packages/app/src/systems/SignUp/machines/signUpMachine.ts

```diff
@@ -146,6 +147,12 @@ export const signUpMachine = createMachine(
         tags: ['loading'],
         invoke: {
           src: 'setupVault',
+          data: {
+            input: (_: MachineContext, ev: any) => ({
+              password: ev.data?.password as string,
```

Comment:
should stop using `as` as a way to force typing, this can bug the application


`as` should be used only in "very" specific cases where we have 100% sure that will not bug anything